procrastinate/aiopg_connector.py

```diff
@@ -166,12 +169,31 @@ async def on_connect(connection):
         final_args.update(pool_args)
         return final_args
 
+    @property
+    def pool(self) -> aiopg.Pool:
+        if self._pool is None:  # Set by open_async
+            raise exceptions.AppNotOpen
```

Comment:
It feels strange to me to raise an `AppNotOpen` exception from `AiopgConnector`.

Reply:
I understand but I think it's the best solution. Raising something more connector-specific means we would have to expect and reraise it in the app and it feels awfully complex.

Reply:
Ok I understand, and it kinda makes sense :-)